### PR TITLE
update 1.9.2.8

### PR DESCRIPTION
**Pull request by redkale**

### CODE REVIEW

**Comment by redkale** [comment]
why not merge

**Comment by nbrady-techempower** [comment]
Added on the weekend. Needs review. Thanks.

File: frameworks/Java/redkale/benchmark_config.json [hunk 1]
```diff
@@ -2,9 +2,25 @@
...
       "plaintext_url": "/plaintext",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Fullstack",
+      "database": "None",
+      "framework": "None",
+      "language": "Java",
+      "flavor": "None",
+      "orm": "Raw",
+      "platform": "Redkale",
```
Comment (msmith-techempower) [inline frameworks/Java/redkale/benchmark_config.json]: `"platform": "None"`

File: frameworks/Java/redkale/benchmark_config.json [hunk 2]
```diff
@@ -2,9 +2,25 @@
...
+      "webserver": "Redkale",
```
Comment (msmith-techempower) [inline frameworks/Java/redkale/benchmark_config.json]: `"webserver": "None"`

**Pull request merged**